app/frontend/src/features/auth/signup/CompleteSignupForm.tsx

```diff
@@ -165,16 +191,33 @@ export default function CompleteSignup() {
           <InputLabel className={authClasses.formLabel} htmlFor="location">
             Your location
           </InputLabel>
-          <TextField
-            id="location"
-            variant="standard"
+          <Controller
             name="location"
-            fullWidth
+            control={control}
             inputRef={register({
               required: "Enter your location",
             })}
```

Comment:
```suggestion
            inputRef={register}
```
That `required` bit isn't doing anything since there's no actual field and you're handling the check manually